src/python/m5/SimObject.py

```diff
@@ -489,6 +489,12 @@ def decorate(func):
                 # We don't cound 'self' as an argument in this case.
                 continue
             param = sig.parameters[param_name]
+            if param.kind in [
+                inspect._ParameterKind.VAR_POSITIONAL,
```

Comment:
According to the documentation, this should be `inspect.Parameter.VAR_POSITIONAL`.

Reply:
Updated the commit. Thanks for providing the correct way to achieve this!